extensions/ql-vscode/src/data-extensions-editor/generate-flow-model.ts

```diff
@@ -0,0 +1,146 @@
+import { CancellationToken } from "vscode";
+import { DatabaseItem } from "../local-databases";
+import { join } from "path";
+import { QueryRunner } from "../queryRunner";
+import { CodeQLCliServer } from "../cli";
+import { extLogger, TeeLogger } from "../common";
+import { definitions } from "./yaml";
+import { ProgressCallback } from "../progress";
+import { getOnDiskWorkspaceFolders } from "../helpers";
+import { ModeledMethod, ModeledMethodType } from "./modeled-method";
+
+class FlowModelGenerator {
+  constructor(
+    private readonly cli: CodeQLCliServer,
+    private readonly queryRunner: QueryRunner,
+    private readonly queryStorageDir: string,
+    private readonly qlDir: string,
+    private readonly databaseItem: DatabaseItem,
+    private readonly progress: ProgressCallback,
+    private readonly token: CancellationToken,
+  ) {}
+
+  async getAddsTo(
```

Comment:
I don't quite understand the name of this function.

Reply:
The name was based on the YAML file which is generated, which has an `addsTo` section. It doesn't make sense anymore since we're now just generating modeled methods, so I'll rename it.